examples/integrations/itk/SelectROI.ipynb

```diff
@@ -0,0 +1,350 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    loaded_scale
</pre><p>Let's make a note, provide a reminder on how the scales, are defined, i.e. "Zero is the highest resolution scale."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/InsightSoftwareConsortium/itkwidgets/pull/703/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Sounds good, updated :+1: